__tests__/app.test.js

```diff
@@ -97,6 +97,7 @@ describe("GET /reviews/:review_id", () => {
           review_img_url: expect.any(String),
           created_at: expect.any(String),
           votes: expect.any(Number),
+          comment_count: expect.any(String),
         });
       });

```

Comment:
Please create a new test to check this feature.

Since we know how many comments the article should have we should test for that exact number instead of any().